packages/menu-button/src/index.js

```diff
@@ -425,6 +425,18 @@ export const MenuItems = forwardRef(function MenuItems(
           }
         } else if (event.key === "Tab") {
           event.preventDefault(); // prevent leaving
+          const nextIndex = state.selectionIndex + 1;
```

Comment:
> Tab: Moves focus to the next element in the tab sequence, and if the item that had focus is not in a menubar, closes its menu and all open parent menu containers.

The next menu item in the list is not the same as the next element in the tab sequence. We don't assume that the item is in a menubar, so ultimately the developer using `MenuButton` would need to add tab behavior if that were the case (at least until we build out a `MenuBar` component one of these days)

Reply:
For standalone menu buttons I believe tab should just close the menu and focus whatever element would come after the menu button in normal tab order.

Reply:
so can you please explain me what do you mean by next element in tab sequence , because originally on press of tab it was working as per your explanation i.e. "For standalone menu buttons I believe tab should just close the menu and focus whatever element would come after the menu button in normal tab order." so if that's the required behavior then I will remove that code